tcks/microprofile-lra/src/main/java/io/quarkus/tck/lra/LRACoordinatorManager.java

```diff
@@ -18,16 +18,18 @@ public class LRACoordinatorManager {
 
     public void beforeClass(
             @Observes(precedence = DEFAULT_PRECEDENCE) org.jboss.arquillian.test.spi.event.suite.BeforeSuite event) {
-        LOGGER.debug("Starting LRA coordinator on port " + coordinatorPort);
-        coordinatorContainer = new GenericContainer<>(DockerImageName.parse("jbosstm/lra-coordinator:latest"))
-                // lra-coordinator is a Quarkus service
-                .withEnv("QUARKUS_HTTP_PORT", String.valueOf(coordinatorPort))
-                // need to run with host network because coordinator calls the TCK services from the container
-                .withNetworkMode("host");
+        if (System.getProperty("lra.coordinator.url") == null) {
+            LOGGER.debug("Starting LRA coordinator on port " + coordinatorPort);
+            coordinatorContainer = new GenericContainer<>(DockerImageName.parse("jbosstm/lra-coordinator:latest"))
+                    // lra-coordinator is a Quarkus service
+                    .withEnv("QUARKUS_HTTP_PORT", String.valueOf(coordinatorPort))
+                    // need to run with host network because coordinator calls the TCK services from the container
+                    .withNetworkMode("host");
```

Comment:
It could add `waitingFor(Wait.forLogMessage(".*lra-coordinator-quarkus.*", 1));` to make sure the server is started.